### PR TITLE
pvs: add page

### PR DESCRIPTION
**Pull request by sahilister**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/pvs.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# pvs
+
+> Display information about physical volumes.
```
Comment (einverne) [inline pages/linux/pvs.md:3]: Is there any links we can put here?

Comment (sahilister) [reply pages/linux/pvs.md:3]: man-pages are the only comprehensive documentation i could find for `pvs`. All the online resources are essentially copying man-pages.

Comment (einverne) [reply pages/linux/pvs.md:3]: Thank you for replying, leave it blank is ok.

File: pages/linux/pvs.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Display only specific fields:
+
+`pvs -o {{field_name_1}},{{field_name_2}}`
+
+- Append field to default display:
+
+`pvs -o +{{field_name}}`
+
+- Suppress heading line:
```
Comment (zdroid) [inline pages/linux/pvs.md:25]:
```suggestion
- Suppress heading lines:
```
Maybe it's better like this?

Comment (sahilister) [reply pages/linux/pvs.md:25]: In `pvs` output, there's always one heading line. Man-page option `--noheadings` option too document it as 'line'.

File: pages/linux/pvs.md (hunk 1 above)
Comment (sbrl) [inline pages/linux/pvs.md:3]: Hey, cool page! I was confused by the description at first though when I tried it for myself, before I looked it up and realised this is to do LVM. Perhaps we could include that here?

```suggestion
> Display information about LVM physical volumes.
```

Comment (sahilister) [reply pages/linux/pvs.md:3]: That's totally justified, I'll soon make a PR adding this.

**Pull request merged**